microsoft/runbook/azure/alma8_gpu_driver.yml

```diff
@@ -0,0 +1,55 @@
+name: azure default
+include:
```

Comment:
We can have a common runbook and pass parameters at runtime. For example: 
```
./lisa.sh -v "resource_group_name:<resource_group_name>"

This will ensure that there are few runbooks which are also easier to maintain. Further, we can also use combinator where you can specify these images and they will be picked at runtime. More info on combinator can be found [here](https://mslisa.readthedocs.io/en/main/write_test/extension.html#combinator)

Reply:
I would like to keep the current multiple runbook yaml files, which gives me an idea of which images and tests we have covered.

Reply:
I think using combinator would be much better. i can give you an example. Can you please tell what variables change in each runbook?